This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marco Borrini <borrinimarco74@gmail.com>
Date: Tue, 4 Aug 2026 15:42:00 +0200
Subject: [PATCH 1/1] Add missing find_dependency in KF6JobWidgetsConfig

When building statically, the CMake targets explicitly declare link
dependencies on KF6Notifications and KF6WidgetsAddons. However, these
packages are not searched for in the Config file. This breaks static
builds of consumers like KIO which cannot find the targets.

diff -ur kjobwidgets-6.28.0/KF6JobWidgetsConfig.cmake.in kjobwidgets-6.28.0-new/KF6JobWidgetsConfig.cmake.in
--- kjobwidgets-6.28.0/KF6JobWidgetsConfig.cmake.in	2026-07-03 12:57:25.000000000 +0200
+++ kjobwidgets-6.28.0-new/KF6JobWidgetsConfig.cmake.in	2026-08-04 15:42:42.260054744 +0200
@@ -5,5 +5,7 @@
 include(CMakeFindDependencyMacro)
 find_dependency(Qt6Widgets @REQUIRED_QT_VERSION@)
 find_dependency(KF6CoreAddons "@KF_DEP_VERSION@")
+find_dependency(KF6Notifications "@KF_DEP_VERSION@")
+find_dependency(KF6WidgetsAddons "@KF_DEP_VERSION@")
 
 include("${CMAKE_CURRENT_LIST_DIR}/KF6JobWidgetsTargets.cmake")
